evm/genesis.py

```diff
@@ -0,0 +1,42 @@
+from eth_utils import decode_hex
+
+from evm.constants import (
+    GENESIS_NONCE,
+    ZERO_HASH32,
+)
+from evm.rlp.headers import BlockHeader
+
+
+MAINNET_GENESIS_HEADER = BlockHeader(
+    difficulty=17179869184,
+    extra_data=decode_hex("0x11bbe8db4e347b4e8c937c1c8370e4b5ed33adb3db69cbdb7a38e1e50b1b82fa"),
+    gas_limit=5000,
+    gas_used=0,
+    bloom=0,
+    mix_hash=ZERO_HASH32,
+    nonce=GENESIS_NONCE,
+    block_number=0,
+    parent_hash=ZERO_HASH32,
+    receipt_root=decode_hex("0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421"),  # noqa: E501
+    uncles_hash=decode_hex("0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347"),  # noqa: E501
+    state_root=decode_hex("0xd7f8974fb5ac78d9ac099b9ad5018bedc2ce0a72dad1827a1709da30580f0544"),
+    timestamp=0,
+    transaction_root=decode_hex("0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421"),  # noqa: E501
+)
+
+ROPSTEN_GENESIS_HEADER = BlockHeader(
+    difficulty=1048576,
+    extra_data=decode_hex("0x3535353535353535353535353535353535353535353535353535353535353535"),
+    gas_limit=16777216,
+    gas_used=0,
+    bloom=0,
+    mix_hash=ZERO_HASH32,
+    nonce=GENESIS_NONCE,
+    block_number=0,
+    parent_hash=ZERO_HASH32,
+    receipt_root=decode_hex("0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421"),  # noqa: E501
+    uncles_hash=decode_hex("0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347"),  # noqa: E501
```

Comment:
This is the same as `evm.constants.EMPTY_UNCLE_HASH` in case you weren't aware that existed.